imageGenSD.py

```diff
@@ -630,30 +661,54 @@ async def showprofile(interaction : dis.Interaction,
         else:
 
             dis_log.debug(f"Creating a SHOW PROFILE view for user {interaction.user.id}.")
-            user_id  = interaction.user.id if user == None else user.id
-            profiles = db_ifc.getProfiles(user_id)
-            dis_log.debug(f"User {user_id} returned profile list {profiles}.")
-            view = ddf.DropdownView(ctx      = interaction,
-                                    type     = ddf.DropDownTypeEnum.SHOW,
-                                    choices  = profiles,
-                                    metadata = metadata)
+            empty_error = "";
+            many_message = "";
 
-            await interaction.response.send_message('Select a profile to view:',view=view)
+            if name == None:
 
-    else:
+                profiles = db_ifc.getUsersProfiles(user_id)
+                empty_error = f"{user_id} does't own any profiles"
+                many_message = f'Select a profile to view:'
 
-        opts = {'id' : profile_id}
+            else:
 
-        dis_log.debug(f"Creating a job with metadata {metadata} and options {opts}.")
-        job = jf.JobFactory.getJob(type    = jf.JobTypeEnum.SHOW_PROFILE,
-                                   ctx     = interaction,
-                                   options = opts)
-        dis_log.debug(f"Posting SHOW job {job} to the queue.")
-        result = show_queue.add(metadata = metadata,
-                                job      = job)
+                dis_log.debug(f"Looking for profiles of name {name}.")
+                profiles = db_ifc.getProfiles(user_id, name)
+                empty_error = f'A character matching the name "{name}" wasn\'t found!'
+                many_message = f'Select a profile with name like "{name}" to view:'
 
-        await interaction.response.send_message(f'{result}', ephemeral=True, delete_after=9.0)
+            if len(profiles) == 0:
+
+                dis_log.debug(f"--Found none {name}")
```

Comment:
Debug message formats should be consistent with other comments (i.e. no '--' in the message).  this occurs in a few other places as well.